quickDeploy.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+INSTALL=false
+MONGOPATH="data/db"
+FIFACHAMPS="$(echo $PWD)"
+STOP=false
+
+for i in "$@"
+do
+  case ${i} in
+    -i|--install)
+      INSTALL=true
+      shift ;;
+    -s|--stop)
+      STOP=true
+      shift ;;
+    --mongo=*)
+      MONGOPATH=`echo $i | sed -e 's/--mongo=//g'`
+      shift ;;
+    *)
+      throw_arg "'${i}' is not a valid argument."
+      shift ;;
+  esac
+done
+
+if [[ ${STOP} == "true" ]]; then
+  kill $(lsof -t -i:4200) > /dev/null 2>&1
```

Comment:
This, I know it kills something, probably the backend or frontend, but it isn't very readable and/or understandable.

Reply:
well.. i believe i can add comments. but shell script is not very human-friendly to read.

Reply:
I can understand it. It kills whatever process is using port 4200 :p

But then again, I LOVE BASH.